zh/deploy-on-gcp-gke.md

```diff
@@ -369,10 +349,7 @@ kubectl --kubeconfig credentials/kubeconfig_<gke_cluster_name> get po -n <tidb_c
 例如，可以将 `tidb_count` 从 1 改为 2 来扩容 TiDB：
 
 ```hcl
-variable "tidb_count" {
-  description = "Number of TiDB nodes per availability zone"
-  default     = 2
-}
+tidb_count = 2
```

Comment:
recommend users to configure via `.tfvars` file

Reply:
Yes, can we make it clear above this block that users can set this in .tfvars?

Reply:
sure, updated